devel/bitwise/Portfile

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        mellowcandle bitwise v0.30
+github.tarball_from releases
+
+name                bitwise
```

Comment:
The `name` is added from the github.setup command and so this line should be removed.

Reply:
Can do. I have seen this is several other existing ports so I emulated it.

Reply:
Done.